packages/eddsa-ticket-pcd/src/EdDSATicketPCD.ts

```diff
@@ -46,53 +52,85 @@ export enum TicketCategory {
  * when verifying tickets.
  */
 export interface ITicketData {
-  // the fields below are not signed and are used for display purposes
-
+  // The fields below are not signed and are used for display purposes.
   attendeeName: string;
   attendeeEmail: string;
   eventName: string;
   ticketName: string;
   checkerEmail: string | undefined;
-
-  // the fields below are signed using the server's private eddsa key
-
-  ticketId: string; // primary key uuid of the ticket's `devconnect_pretix_tickets` entry
-  eventId: string; // primary key uuid of the event's `pretix_events_config` entry
-  productId: string; // primary key uuid of the ticket's `devconnect_pretix_items_info` entry
+  // The fields below are signed using the server's private eddsa key.
```

Comment:
```suggestion
  // The fields below are signed using the Zupass server's private eddsa key.
```

Reply:
@ichub Shouldn't this package be independent of Zupass? Ar are the parameters so specific that they can only be used on Zupass?